task-2-SERVLETS/Self-check.md

```diff
@@ -0,0 +1,13 @@
+1. What is the Servlet Container? List some of them.
+2. What is the life-cycle of a servlet?
+3. What are life-cycle methods for a servlet do you know and what is their functions?
+4. What is REST?
+5. Which HTTP methods are used according to the REST?
+6. Describe which http-statuses are used in responses and what kind of server behavior do they mean according to the REST.
+7. What is the difference between PUT and PATCH methods?
+8. What is the Servlet Listener? What is the purpose of it?
+9. What is the Servlet Filter? What is the purpose of it?
+10. Describe a server behaviour when a request is sent to the server?
+11. What is the JSP? And what is the difference to HTML?
```

Comment:
```suggestion
11. What is the JSP? And what is the difference with HTML?
```

Reply:
According to British English, this option is correct. But for more standard view we can also use 'from'
[link](https://getproofed.com/writing-tips/word-choice-different-than-different-to-or-different-from/)